### PR TITLE
Add ML methodology

### PR DESCRIPTION
**Pull request by vmarkovtsev**
Signed-off-by: Vadim Markovtsev <vadim@sourced.tech>

### CODE REVIEW

File: engineering/teams/methodology-ml.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- All the team is running on Ubuntu. You can use other OS if you like adventures and want to spend time on solving unavoidable compatibility problems.
+- We ping the Lead or the maintainer every few hours in DM if there is a review-pending PR which is blocking.
+- We do Research Days, see the corresponding section below for details.
+- We write our papers in [Overleaf](https://overleaf.com) and post them to 🔑[src-d/papers](https://github.com/src-d/papers) once written.
+- We describe our bright ideas in 🔑[src-d/feature-idea](https://github.com/src-d/feature-idea).
+- We do not forget to ask the Lead for extra days off if we had to work on the weekends.
+
+## Interviews
+
+This part cannot is classified, unfortunately. It should describe our ML interview process. Refer to 🔑[src-d/company](https://github.com/src-d/company/tree/master/interviews/engineering).
```
Comment (Jan21) [inline engineering/teams/methodology-ml.md]: "cannot" should be deleted here.

File: engineering/teams/methodology-ml.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Machine Learning (ML) team follows the [common methodology](https://github.com/src-d/guide/blob/master/engineering/methodology.md) shared by
+the rest of the teams, with the following additions:
+
+- We meet two times per week, on Tuesday and on Thursday, to sync and discuss the status.
```
Comment (m09) [inline engineering/teams/methodology-ml.md]:
```suggestion
- We meet twice a week, on Tuesday and on Thursday, to sync and discuss the status of current work.
```

File: engineering/teams/methodology-ml.md (hunk 1 above)
Comment (m09) [inline engineering/teams/methodology-ml.md]:
```suggestion
This part is classified, unfortunately. It should describe our ML interview process. Refer to 🔑[src-d/company](https://github.com/src-d/company/tree/master/interviews/engineering).
```

File: engineering/teams/methodology-ml.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- We do not forget to ask the Lead for extra days off if we had to work on the weekends.
+
+## Interviews
+
+This part cannot is classified, unfortunately. It should describe our ML interview process. Refer to 🔑[src-d/company](https://github.com/src-d/company/tree/master/interviews/engineering).
+
+## Research Day
+
+Once per two weeks, on Fridays, the same day as the [paper reading club](https://github.com/src-d/reading-club),
+we run Research Days (RD). The purpose of RDs is to stay aware of the latest research and encourage experimentats with MLonCode.
```
Comment (m09) [inline engineering/teams/methodology-ml.md]:
```suggestion
we run Research Days (RD). The purpose of RDs is to stay aware of the latest research and encourage experiments with MLonCode.
```

File: engineering/teams/methodology-ml.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+the rest of the teams, with the following additions:
+
+- We meet two times per week, on Tuesday and on Thursday, to sync and discuss the status.
+- We meet once per month to do middle-term planning of the [OKRs](https://github.com/src-d/okrs) fulfillment. We fuse that meeting with a retrospective.
+- We minute every meeting, whether it is with people from academia or internal. The up-to-date order of scribes is in [ml-backlog](https://github.com/src-d/ml-backlog#minutes-scribe-order), they are chosen round robin.
+- We post our [OSD](../../talent/open_source_days.md) plans to 🔑`#osd` channel on Slack in the morning.
+- We mention the URLs of the issues/PRs we work on in our daily Geekbot standups.
+- We passionately, thoughtfully follow the [Python style guide](../conventions/python.md).
+- We do **not** use Anaconda.
+- All the team is running on Ubuntu. You can use other OS if you like adventures and want to spend time on solving unavoidable compatibility problems.
```
Comment (zurk) [inline engineering/teams/methodology-ml.md]: I run on MacOS and Ubuntu both and everything is perfect :)

File: engineering/teams/methodology-ml.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- We ping the Lead or the maintainer every few hours in DM if there is a review-pending PR which is blocking.
+- We do Research Days, see the corresponding section below for details.
+- We write our papers in [Overleaf](https://overleaf.com) and post them to 🔑[src-d/papers](https://github.com/src-d/papers) once written.
```
Comment (zurk) [inline engineering/teams/methodology-ml.md]:
```suggestion
- We write our papers in [Overleaf](https://overleaf.com) and post them to 🔑[src-d/papers](https://github.com/src-d/papers) once written.
- We put all the code related to the paper only (plot generation, stats computation, etc) into 🔑[src-d/papers](https://github.com/src-d/papers)
```

File: engineering/teams/methodology-ml.md (hunk 4 above)
Comment (vmarkovtsev) [reply engineering/teams/methodology-ml.md]: This was not the case with @irinakhismatullina, I still remember those weird problems with Babelfish she had.

File: engineering/teams/methodology-ml.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## Interviews
+
+This part is classified, unfortunately. It should describe our ML interview process. Refer to 🔑[src-d/company](https://github.com/src-d/company/tree/master/interviews/engineering).
+
+## Research Day
+
+Once per two weeks, on Fridays, the same day as the [paper reading club](https://github.com/src-d/reading-club),
+we run Research Days (RD). The purpose of RDs is to stay aware of the latest research and encourage experiments with MLonCode.
+Here is the list of things which may be done on a RD:
```
Comment (smola) [inline engineering/teams/methodology-ml.md]:
```suggestion
Here is the list of things which may be do on a RD:
```

Comment (vmarkovtsev) [reply engineering/teams/methodology-ml.md]: @smola :thinking: ?

**Comment by vmarkovtsev** [comment]
@smola good to merge?

**Pull request merged**